src/@chakra-ui/gatsby-plugin/foundations/colors.ts

```diff
@@ -10,30 +10,35 @@ const colors = {
     900: "#141414",
   },
   blue: {
-    100: "#e8e8ff",
-    300: "#8d8dff",
+    100: "#dedeff",
+    300: "#ababfe",
     500: "#1c1cff",
+    600: "#090990",
     700: "#0b0b66",
   },
   orange: {
     100: "#ffe3d3",
     300: "#ffb991",
     500: "#ff7324",
-    800: "#451900",
+    600: "#c95d20",
+    800: "#6a3301",
   },
   red: {
-    100: "#F7C8C8",
+    100: "#f7c8c8",
     500: "#b80000",
     900: "#1B0C0C",
   },
   green: {
-    100: "#C8F7D8",
+    100: "#ddf4e4",
+    // ! Deprecating 400
     400: "#48BB78",
-    500: "#109e62",
+    500: "#0a7146",
+    // ! Deprecating 900
     900: "#0A160E",
   },
   yellow: {
     200: "#fff8df",
+    500: "#bd8400",
   },
 }
```

Comment:
I think we should also add the brand colors: https://www.figma.com/file/NrNxGjBL0Yl1PrNrOT8G2B/ethereum.org-Design-System?node-id=71%3A516

Not sure if we should follow the `{colorName}.{scaleNumber}` convention here. Maybe we could add it like this:
```tsx
const colors = {
  brand: {
    jordyBlue: "...",
    pantoneBlue: "...",
    ....
  }
}
```
what do you think?

Reply:
From Ethiopia I'm sintayew Gashaw

Reply:
@nloureiro not sure. Are we planning on using those colors on other parts of the site?

- If **no** then I wouldn't add them on the theme yet.
- If **yes** then I would add them as I mention previously. Following the same name you used on the DS.

Would be something like

```tsx
const colors = {
  gray: {
    100: "#f7f7f7",
    ....
  },
  blue: {
    ...
  },
  orange: {
    ....
  },
  // the 5 brand colors
  brand: {
    jordyBlue: "...",
    celesteGreen: "...",
    babyPink: "..."
  }
}
```

The naming for those colors would be the names you used on the DS
![image](https://user-images.githubusercontent.com/468158/228375489-19b56f31-c11a-4857-a690-8ab52254f14f.png)